src/app/forms/templates/general/adr.template.ts

```diff
@@ -491,13 +500,56 @@ export const AdrTemplate: FormNode = {
       viewComponent: AdrTankStatementUnNumberViewComponent,
       groups: ['productList', 'statement_select_hide', 'tank_details_hide', 'dangerous_goods'],
       hide: true,
-      customErrorMessage: 'Reference number or UN number 1 is required when selecting Product List',
       validators: [
         {
-          name: ValidatorNames.RequiredIfEquals,
-          args: { sibling: 'techRecord_adrDetails_tank_tankDetails_tankStatement_productListRefNo', value: [null, undefined, ''] },
+          name: ValidatorNames.Custom,
+          args: (control: CustomFormControl) => {
+            if (control.meta?.hide) {
```

Comment:
is there a better place for this logic to sit instead of inside of the template? feels like this breaks the nice readable flow that is in place and if we start adding similar solutions to this it could get quite cluttered